packages/router/test-setup.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable global-require,no-plusplus */
+import 'babel-polyfill';
```

Comment:
This needs to be in package.json
This should be spotted by linter, can you add the module in travis linter part ?